### ISSUE TITLE
Blocked messages don't get logged.

### ISSUE THREAD

**Issue by Mm2PL** [Chatterino/chatterino2#2228]
**Describe the bug**

No messages blocked by regexes will appear in the logs

**To reproduce**

  1. Add a blocked phrase. With a config something like this:
    - Pattern: `my phrase`
    - Case sensitive: yes
    - Block: **yes**
  2. Make a bot or another user say the blocked phrase.
  3. Observe that it doesn't appear in the logs.

**Screenshots**

N/A

**Chatterino version**

05aab868 + patches irrelevant to this issue

**Operating system**

Ubuntu 20.04

**Additional information**

As expected the message doesn't show in Chatterino.

**Comment by Felanbird** [comment]
Partial Duplicate of #1330 

same concept I mentioned in that issue, just with blocked messages instead of ignoring

**Comment by pajlada** [comment]
Same close reason as #1330

As a chat client, Chatterino is logging what's printed on the screen.
Not a bug, won't fix.
